packages/standard-minifier-js/plugin/minify-js.js

```diff
@@ -1,161 +1,105 @@
+          data: file.getContentsAsBuffer(),
+          sourceMap: file.getSourceMap(),
+          path: file.getPathInBundle(),
+        });
       });
-    });
-    return;
-  }
-
-  function maybeThrowMinifyErrorBySourceFile(error, file) {
-    var minifierErrorRegex = /^(.*?)\s?\((\d+):(\d+)\)$/;
-    var parseError = minifierErrorRegex.exec(error.message);
-
-    if (!parseError) {
-      // If we were unable to parse it, just let the usual error handling work.
-      return;
-    }
-
-    var lineErrorMessage = parseError[1];
-    var lineErrorLineNumber = parseError[2];
-
-    var parseErrorContentIndex = lineErrorLineNumber - 1;
-
-    // Unlikely, since we have a multi-line fixed header in this file.
-    if (parseErrorContentIndex < 0) {
-      return;
-    }
-
-    /*
-
-    What we're parsing looks like this:
-
-    /////////////////////////////////////////
-    //                                     //
-    // path/to/file.js                     //
-    //                                     //
-    /////////////////////////////////////////
-                                           // 1
-       var illegalECMAScript = true;       // 2
-                                           // 3
-    /////////////////////////////////////////
-
-    Btw, the above code is intentionally not newer ECMAScript so
-    we don't break ourselves.
-
-    */
-
-    var contents = file.getContentsAsString().split(/\n/);
-    var lineContent = contents[parseErrorContentIndex];
-
-    // Try to grab the line number, which sometimes doesn't exist on
-    // line, abnormally-long lines in a larger block.
-    var lineSrcLineParts = /^(.*?)(?:\s*\/\/ (\d+))?$/.exec(lineContent);
-
-    // The line didn't match at all?  Let's just not try.
-    if (!lineSrcLineParts) {
       return;
     }
 
-    var lineSrcLineContent = lineSrcLineParts[1];
-    var lineSrcLineNumber = lineSrcLineParts[2];
-
-    // Count backward from the failed line to find the filename.
-    for (var c = parseErrorContentIndex - 1; c >= 0; c--) {
-      var sourceLine = contents[c];
-
-      // If the line is a boatload of slashes, we're in the right place.
-      if (/^\/\/\/{6,}$/.test(sourceLine)) {
-
-        // If 4 lines back is the same exact line, we've found the framing.
-        if (contents[c - 4] === sourceLine) {
-
-          // So in that case, 2 lines back is the file path.
-          var parseErrorPath = contents[c - 2]
-            .substring(3)
-            .replace(/\s+\/\//, "");
-
-          var minError = new Error(
-            "Babili minification error " +
-            "within " + file.getPathInBundle() + ":\n" +
-            parseErrorPath +
-            (lineSrcLineNumber ? ", line " + lineSrcLineNumber : "") + "\n" +
-            "\n" +
-            lineErrorMessage + ":\n" +
-            "\n" +
-            lineSrcLineContent + "\n"
-          );
-
-          throw minError;
-        }
+    // this function tries its best to locate the original source file
```

Comment:
this function needed to be changed to account for the error.message field being different between babel-minify and terser, but I made a little driver program to test this function inside of and used some bundled files from a meteor build that i wrote to disk as test data and it worked well during testing